projects/plugins/inspect/package.json

```
@@ -0,0 +1,49 @@
+{
+	"private": true,
+	"name": "@automattic/jetpack-inspect",
+	"version": "0.1.0-alpha",
+	"description": "A debugging plugin to inspect Jetpack interactions with WordPress.com.",
+	"homepage": "https://jetpack.com",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[Plugin] Inspect"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/plugins/inspect"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build": "echo 'Not implemented.'",
+		"build-js": "echo 'Not implemented.'",
+		"build-production": "echo 'Not implemented.'",
+		"build-production-js": "echo 'Not implemented.'",
+		"clean": "true"
+	},
+	"engines": {
+		"node": "^18.13.0",
+		"yarn": "use pnpm instead - see docs/yarn-upgrade.md"
+	},
+	"dependencies": {
+		"svelte": "3.58.0",
+		"zod": "3.20.2"
+	},
+	"devDependencies": {
+		"@babel/core": "7.21.5",
+		"@babel/preset-env": "7.21.5",
+		"@rollup/plugin-alias": "5.0.0",
+		"@rollup/plugin-commonjs": "19.0.2",
+		"@rollup/plugin-node-resolve": "13.3.0",
+		"@rollup/plugin-typescript": "8.3.3",
+		"@wordpress/i18n": "4.35.0",
+		"postcss": "8.4.21",
+		"rollup": "2.79.1",
+		"rollup-plugin-livereload": "2.0.5",
+		"rollup-plugin-postcss": "4.0.2",
+		"rollup-plugin-svelte": "7.1.0",
+		"rollup-plugin-terser": "7.0.2",
+		"svelte-preprocess": "5.0.3",
+		"typescript": "5.0.4"
+	}
```

Two changes here:
* `rollup-plugin-terser` should be replaced by `@rollup/plugin-terser`
* Adding in the optional peer dep on `tslib` avoids an extra entry in pnpm-lock.

```suggestion
	"devDependencies": {
		"@babel/core": "7.21.5",
		"@babel/preset-env": "7.21.5",
		"@rollup/plugin-alias": "5.0.0",
		"@rollup/plugin-commonjs": "19.0.2",
		"@rollup/plugin-node-resolve": "13.3.0",
		"@rollup/plugin-terser": "0.4.3",
		"@rollup/plugin-typescript": "8.3.3",
		"@wordpress/i18n": "4.35.0",
		"postcss": "8.4.21",
		"rollup": "2.79.1",
		"rollup-plugin-livereload": "2.0.5",
		"rollup-plugin-postcss": "4.0.2",
		"rollup-plugin-svelte": "7.1.0",
		"svelte-preprocess": "5.0.3",
		"tslib": "2.5.0",
		"typescript": "5.0.4"
	}
```